components/Contributors/OscrPill.tsx

```diff
@@ -2,18 +2,18 @@ import Pill from "components/atoms/Pill/pill";
 import Tooltip from "components/atoms/Tooltip/tooltip";
 
 export const OscrPill = ({ rating, hideRating }: { rating: number | undefined; hideRating: boolean }) => {
-  let percentageRating = rating ? Math.floor(rating * 100) : 0;
+  let ratingToDisplay = rating ? Math.floor(rating * 100) : 0;
 
-  if (percentageRating < 1) {
-    percentageRating = 0;
+  if (ratingToDisplay < 1) {
+    ratingToDisplay = 0;
   }
 
   return (
     <Tooltip direction="top" content="Open Source Contributor Rating (OSCR)">
       {hideRating ? (
         <Pill color="purple" size="small" text="00" blurText={true} />
       ) : (
-        <Pill color="purple" size="small" text={`${percentageRating}`} />
+        <>{ratingToDisplay > 0 ? <Pill color="purple" size="small" text={`${ratingToDisplay}`} /> : <span>-</span>}</>
```

Comment:
This isn't quite right: 0 **_is_** a valid OSCR, especially if someone hasn't done anything recently or they've committed one of the cardinal sins (like having a PR closed as spam which deducts -99 and will typically completely reset their score to zero).

Can we leverage `devstats_updated_at` which gets updated on the user entity when the oscr is re-calculated? If the date is a nullish date of `1970-01-01 00:00:00`?

Reply:
I agree 0 is valid. 

My thought process for the change is seeing a lot of 0's in succession. 

I walk back my "not enough info" comment from slack. That is a different scenario. 

FWIW. I am fine abandon this and opt for not scope creeping ourselves.